src/buttons/src/Button.js

```diff
@@ -139,7 +139,7 @@ Button.propTypes = {
   /**
    * The appearance of the button.
    */
-  appearance: PropTypes.oneOf(['default', 'minimal', 'primary']),
+  appearance: PropTypes.string,
```

Comment:
@zkuzmic @kamebkj @akleiner2 This change makes sense to me until we iron out the typed theming support more, similar to the change for `intent` -> `string` in #1341. @CLSnazel Would you mind updating the `ButtonAppearance` type alias in `index.d.ts` to just `string` as well?

```ts
export type ButtonAppearance = string
```

Reply:
Thank you for that pointer! `index.d.ts` has been updated accordingly. Please let me know if there's any additional changes required.